ci-automation/push_pkgs.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+#
+# Copyright (c) 2021 The Flatcar Maintainers.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# >>> This file is supposed to be SOURCED from the repository ROOT. <<<
+#
+# push_packages() should be called w/ the positional INPUT parameters below.
+
+# OS image binary packages publisher automation stub.
+#   This script will publish the packages from a pre-built packages container to
+#   the buildcache server, effectively turning the build cache into a
+#   binary packages server for the SDK.
+#
+# PREREQUISITES:
+#
+#   1. SDK version and OS image version are recorded in sdk_container/.repo/manifests/version.txt
+#   2. Scripts repo version tag of OS image version to be built is available and checked out.
+#   3. Flatcar packages container is available via build cache server
+#       from "/containers/[VERSION]/flatcar-packages-[ARCH]-[FLATCAR_VERSION].tar.gz"
+#       or present locally. Container must contain binary packages and torcx artefacts.
```

Comment:
Kuberentes and artefact, our favourite types. ;)

```suggestion
#       or present locally. Container must contain binary packages and torcx artifacts.
```

Reply:
OH NO - my nemesis 'artefacts' strikes again! Thanks for catching :)